src/Service/Operation/ListOrganismeOperation.php

```diff
@@ -14,7 +14,7 @@ public function __construct(
     /**
      * @return array<Organisme>
      */
-    public function listOrganismes(string $type = 'Z'): array
+    public function listOrganismes(string $type): array
```

Comment:
This is a BC break: argument is now required.

Can you explain why this change is needed ?